tests/test_driver_get_or_create.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import unittest
+import sys
+from unittest.mock import Mock
+
+from gpiozero.pins.mock import MockFactory
+
+# pylint: disable=wrong-import-position
+from octoprint_octorelay.driver import Relay
+
+# avoid keeping other modules automatically imported by this test
+del sys.modules["octoprint_octorelay"]
+del sys.modules["octoprint_octorelay.migrations"]
+del sys.modules["octoprint_octorelay.task"]
+
+class TestRelayDriver(unittest.TestCase):
+    def test_get(self):
```

Comment:
can these tests be moved into the existing `test_driver.py`?

Reply:
I also think it can be reduced into one test by just making the second relay inverted

```py
        relay2 = Relay.get_or_create_relay(20, True, MockFactory()) # inverted
```